### ISSUE TITLE
Different region setting for virtual desktop dont work

### ISSUE THREAD

**Issue by Eugene847** [digitalcreations/MaxTo#669]
When I try to set different region on the virtual desktop as for the 1st desktop It dont work. 
Example: 
1st desktop should have two regions with 40%/60% horicontal split. 
2nd desktop (virtual) should have three regions with other splits of each region. 

I use only one physical monitor.
In the settings > monitor > virtual desktops the setting "use different regions on any virtual desktops" (translate from my native speech into english) is checked. I also un-checked and make the test.
I also restarted the systems several times but the situation keep the same.

**Comment by vegardlarsen** [comment]
Can you please attach your log files? `%LocalAppData%\MaxTo`

**Comment by Eugene847** [comment]
Hi, what is the name of the log files, in the folder I can not find *.log file or something what looked like a log file.
Thanks for help

**Comment by vegardlarsen** [comment]
I apologize. I meant to type `%AppData%\MaxTo`. Sorry about that. :)

**Comment by Eugene847** [comment]
Hi, the log contains some data, I dont want to share (business email, filenames i.e.). I try to create a new log and come back

**Comment by gsrodrigostadler** [comment]
I am able to share my log file, maybe that will help, since I have the same problem. I am not sure if it was the update to Windows 10 20H2 or the update to MaxTo 2.1, since I did both the same day.

Log File:
```
2020-11-27 07:23:57 [Core@2.1.4.725] [Information] [] Logger setup (online logging: True, process #15252)
2020-11-27 07:23:58 [Core@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:23:59 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #15252)
2020-11-27 07:23:59 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #15252)
2020-11-27 07:23:59 [Core@2.1.4.725] [Information] [MaxTo.Core.Shims.ChromiumHookBlockInitializer] Creating registry keys to unblock hooks for Chromium-based browsers: "Edge Dev, Edge Beta, Chromium, Google Chrome, Brave, Iridium".
2020-11-27 07:24:01 [Core@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:24:01 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Squirrel-installed app, so rewriting CLI batch file.
2020-11-27 07:24:01 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Checking for updates.
2020-11-27 07:24:02 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Found no new updates.
2020-11-27 07:24:02 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] No updates found
2020-11-27 07:24:02 [Server@2.1.4.725] [Information] [] Logger setup (online logging: True, process #2788)
2020-11-27 07:24:03 [Core@2.1.4.725] [Information] [] Server up and running, client connected.
2020-11-27 07:24:03 [Core@2.1.4.725] [Information] [MaxTo.Core.Bootstrapper] 4 processes started
2020-11-27 07:24:03 [CompanionX64@2.1.4.725] [Information] [] Logger setup (online logging: True, process #1044)
2020-11-27 07:24:03 [CompanionX64@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:24:04 [CompanionX86@2.1.4.725] [Information] [] Logger setup (online logging: True, process #29540)
2020-11-27 07:24:08 [CompanionX64@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:24:12 [CompanionX86@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:24:13 [CompanionX64@2.1.4.725] [Information] [MaxTo.Companion.Hook.HookController] Enabled hook
2020-11-27 07:24:21 [CompanionX64@2.1.4.725] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "New Tab - Google Chrome" (0x"104b6"), used mouse position to find region "{X=0,Y=0,Width=1920,Height=1050}".
2020-11-27 07:24:48 [CompanionX64@2.1.4.725] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "Microsoft Teams" (0x"104a8"), used mouse position to find region "{X=2880,Y=0,Width=960,Height=1050}".
2020-11-27 07:24:50 [CompanionX86@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:24:52 [CompanionX86@2.1.4.725] [Information] [MaxTo.Companion.Hook.HookController] Enabled hook
2020-11-27 07:24:58 [UserInterface@2.1.4.725] [Information] [] Logger setup (online logging: True, process #11204)
2020-11-27 07:25:00 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.ViewModels.NotificationIconViewModel"
2020-11-27 07:25:10 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.SettingsViewModel"
2020-11-27 07:25:11 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.Recipes.Index.IndexViewModel"
2020-11-27 07:25:11 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.Recipes.RecipesViewModel"
2020-11-27 07:25:12 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.Index.IndexViewModel". Closed: False
2020-11-27 07:25:12 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.RecipesViewModel". Closed: False
2020-11-27 07:25:12 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.About.AboutViewModel"
2020-11-27 07:25:14 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.Index.IndexViewModel". Closed: True
2020-11-27 07:25:14 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.RecipesViewModel". Closed: True
2020-11-27 07:25:14 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.About.AboutViewModel". Closed: True
2020-11-27 07:25:14 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.SettingsViewModel". Closed: True
2020-11-27 07:26:07 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:26:10 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
2020-11-27 07:26:14 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:26:18 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
2020-11-27 07:26:22 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.About.AboutViewModel"
2020-11-27 07:26:22 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.About.AboutViewModel". Closed: False
2020-11-27 07:26:22 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.SettingsViewModel"
2020-11-27 07:26:22 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.Recipes.Index.IndexViewModel"
2020-11-27 07:26:22 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.Recipes.RecipesViewModel"
2020-11-27 07:26:25 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.Index.IndexViewModel". Closed: False
2020-11-27 07:26:25 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.RecipesViewModel". Closed: False
2020-11-27 07:26:25 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.Components.Settings.Monitor.MonitorViewModel"
2020-11-27 07:26:26 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #15252)
2020-11-27 07:26:26 [Core@2.1.4.725] [Information] [MaxTo.Core.Keyboard.HotkeyRegistrationService] Re-registering hotkeys because configuration changed
2020-11-27 07:26:28 [Core@2.1.4.725] [Information] [MaxTo.Core.Keyboard.HotkeyRegistrationService] Re-registering hotkeys because configuration changed
2020-11-27 07:26:31 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Recipes.RecipesViewModel". Closed: True
2020-11-27 07:26:31 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.Monitor.MonitorViewModel". Closed: True
2020-11-27 07:26:31 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.About.AboutViewModel". Closed: True
2020-11-27 07:26:31 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.Components.Settings.SettingsViewModel". Closed: True
2020-11-27 07:26:34 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:26:44 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
2020-11-27 07:26:49 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:26:51 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
2020-11-27 07:28:52 [Server@2.1.4.725] [Information] [MaxTo.Server.ServerApplicationContext] Received exit message. Exiting in 250ms...
2020-11-27 07:28:52 [Core@2.1.4.725] [Information] [MaxTo.Core.Bootstrapper] Received exit signal, waiting for subprocesses to shut down
2020-11-27 07:28:52 [UserInterface@2.1.4.725] [Information] [] Deactivated "MaxTo.UI.ViewModels.NotificationIconViewModel". Closed: True
2020-11-27 07:28:52 [Server@2.1.4.725] [Warning] [MaxTo.Core.Communication.NamedPipeServerManager] Lost connection to "61eb6e15-f6d2-42cc-a9ae-b8e60161312f"
System.IO.IOException: Pipe is broken.
   at System.IO.Pipes.PipeStream.WinIOError(Int32 errorCode)
   at System.IO.Pipes.PipeStream.BeginWriteCore(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   at System.IO.Stream.<>c.<BeginEndWriteAsync>b__53_0(Stream stream, ReadWriteParameters args, AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncTrim[TInstance,TArgs](TInstance thisRef, TArgs args, Func`5 beginMethod, Func`3 endMethod)
   at System.IO.Stream.BeginEndWriteAsync(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.Stream.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at MaxTo.Core.Communication.NamedPipeServer.<Send>d__26.MoveNext()
2020-11-27 07:28:52 [Server@2.1.4.725] [Warning] [MaxTo.Core.Communication.NamedPipeServerManager] Lost connection to "c1d85c42-ac65-4d14-af10-03720602397b"
System.IO.IOException: Pipe is broken.
   at System.IO.Pipes.PipeStream.WinIOError(Int32 errorCode)
   at System.IO.Pipes.PipeStream.BeginWriteCore(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   at System.IO.Stream.<>c.<BeginEndWriteAsync>b__53_0(Stream stream, ReadWriteParameters args, AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncTrim[TInstance,TArgs](TInstance thisRef, TArgs args, Func`5 beginMethod, Func`3 endMethod)
   at System.IO.Stream.BeginEndWriteAsync(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.Stream.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at MaxTo.Core.Communication.NamedPipeServer.<Send>d__26.MoveNext()
2020-11-27 07:28:52 [CompanionX64@2.1.4.725] [Information] [MaxTo.Companion.Hook.HookController] Disabled hook because of exit
2020-11-27 07:28:52 [CompanionX86@2.1.4.725] [Information] [MaxTo.Companion.Hook.HookController] Disabled hook because of exit
2020-11-27 07:28:52 [Server@2.1.4.725] [Warning] [MaxTo.Core.Communication.NamedPipeServerManager] Lost connection to "b7bbed1c-392c-4128-97e7-b23ae765847d"
System.IO.IOException: Pipe is broken.
   at System.IO.Pipes.PipeStream.WinIOError(Int32 errorCode)
   at System.IO.Pipes.PipeStream.BeginWriteCore(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   at System.IO.Stream.<>c.<BeginEndWriteAsync>b__53_0(Stream stream, ReadWriteParameters args, AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncTrim[TInstance,TArgs](TInstance thisRef, TArgs args, Func`5 beginMethod, Func`3 endMethod)
   at System.IO.Stream.BeginEndWriteAsync(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.Stream.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at MaxTo.Core.Communication.NamedPipeServer.<Send>d__26.MoveNext()
2020-11-27 07:28:52 [Server@2.1.4.725] [Warning] [MaxTo.Core.Communication.NamedPipeServerManager] Lost connection to "b090755a-b1a1-4d2b-9f69-5d18c410e431"
System.IO.IOException: Pipe is broken.
   at System.IO.Pipes.PipeStream.WinIOError(Int32 errorCode)
   at System.IO.Pipes.PipeStream.BeginWriteCore(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   at System.IO.Stream.<>c.<BeginEndWriteAsync>b__53_0(Stream stream, ReadWriteParameters args, AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncTrim[TInstance,TArgs](TInstance thisRef, TArgs args, Func`5 beginMethod, Func`3 endMethod)
   at System.IO.Stream.BeginEndWriteAsync(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.Stream.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at MaxTo.Core.Communication.NamedPipeServer.<Send>d__26.MoveNext()
2020-11-27 07:28:52 [Server@2.1.4.725] [Warning] [MaxTo.Core.Communication.NamedPipeServerManager] Lost connection to "e39f91f8-9b82-4168-987a-cc77a11e2acf"
System.IO.IOException: Pipe is broken.
   at System.IO.Pipes.PipeStream.WinIOError(Int32 errorCode)
   at System.IO.Pipes.PipeStream.BeginWriteCore(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   at System.IO.Stream.<>c.<BeginEndWriteAsync>b__53_0(Stream stream, ReadWriteParameters args, AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncTrim[TInstance,TArgs](TInstance thisRef, TArgs args, Func`5 beginMethod, Func`3 endMethod)
   at System.IO.Stream.BeginEndWriteAsync(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.Stream.WriteAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at MaxTo.Core.Communication.NamedPipeServer.<Send>d__26.MoveNext()
2020-11-27 07:28:55 [Core@2.1.4.725] [Information] [] Logger setup (online logging: True, process #22392)
2020-11-27 07:28:55 [Core@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:28:56 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #22392)
2020-11-27 07:28:56 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #22392)
2020-11-27 07:28:56 [Core@2.1.4.725] [Information] [MaxTo.Core.Shims.ChromiumHookBlockInitializer] Creating registry keys to unblock hooks for Chromium-based browsers: "Edge Dev, Edge Beta, Chromium, Google Chrome, Brave, Iridium".
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Squirrel-installed app, so rewriting CLI batch file.
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Checking for updates.
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Found no new updates.
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] No updates found
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [] Was started as normal user, expected elevated. Restarting.
2020-11-27 07:28:58 [Core@2.1.4.725] [Information] [MaxTo.Core.RestartService] Restarting MaxTo with arguments "--elevated --waitforexit 22392"
2020-11-27 07:29:01 [Core@2.1.4.725] [Information] [] Logger setup (online logging: True, process #22512)
2020-11-27 07:29:01 [Core@2.1.4.725] [Information] [] Process #22392 was not running, continuing with startup.
2020-11-27 07:29:01 [Core@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:29:02 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #22512)
2020-11-27 07:29:02 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #22512)
2020-11-27 07:29:02 [Core@2.1.4.725] [Information] [MaxTo.Core.Shims.ChromiumHookBlockInitializer] Creating registry keys to unblock hooks for Chromium-based browsers: "Edge Dev, Edge Beta, Chromium, Google Chrome, Brave, Iridium".
2020-11-27 07:29:03 [Core@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:29:03 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Squirrel-installed app, so rewriting CLI batch file.
2020-11-27 07:29:03 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Checking for updates.
2020-11-27 07:29:04 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] Found no new updates.
2020-11-27 07:29:04 [Core@2.1.4.725] [Information] [MaxTo.Core.Update.UpdateService] No updates found
2020-11-27 07:29:04 [Server@2.1.4.725] [Information] [] Logger setup (online logging: True, process #29884)
2020-11-27 07:29:04 [Core@2.1.4.725] [Information] [] Server up and running, client connected.
2020-11-27 07:29:05 [Core@2.1.4.725] [Information] [MaxTo.Core.Bootstrapper] 4 processes started
2020-11-27 07:29:05 [CompanionX86@2.1.4.725] [Information] [] Logger setup (online logging: True, process #17316)
2020-11-27 07:29:05 [CompanionX64@2.1.4.725] [Information] [] Logger setup (online logging: True, process #22804)
2020-11-27 07:29:05 [CompanionX86@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:29:05 [CompanionX64@2.1.4.725] [Information] [MaxTo.Core.Settings.DefaultConfigurationLocation] Found no specified configuration location file, using default settings location "C:\Users\rodrigostadler\AppData\Roaming\MaxTo"
2020-11-27 07:29:07 [CompanionX64@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:29:07 [CompanionX64@2.1.4.725] [Information] [MaxTo.Companion.Hook.HookController] Enabled hook
2020-11-27 07:29:07 [CompanionX86@2.1.4.725] [Information] [] Read shims from "C:\Users\rodrigostadler\AppData\Local\MaxTo\shims.json"
2020-11-27 07:29:08 [CompanionX86@2.1.4.725] [Information] [MaxTo.Companion.Hook.HookController] Enabled hook
2020-11-27 07:29:08 [UserInterface@2.1.4.725] [Information] [] Logger setup (online logging: True, process #24764)
2020-11-27 07:29:09 [UserInterface@2.1.4.725] [Information] [] Activated "MaxTo.UI.ViewModels.NotificationIconViewModel"
2020-11-27 07:29:09 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #22512)
2020-11-27 07:29:09 [Core@2.1.4.725] [Information] [] Logger setup (online logging: False, process #22512)
2020-11-27 07:29:15 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:29:17 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
2020-11-27 07:29:20 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:29:23 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
2020-11-27 07:29:28 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Found 1 screens.
2020-11-27 07:29:31 [UserInterface@2.1.4.725] [Information] [MaxTo.UI.ViewModels.SetRegionsWindowManager] Updated regions on "HTJM302347"
```

**Comment by vegardlarsen** [comment]
@gsrodrigostadler Unfortunately, the log file does not contain any related messages.

I get suspicious when you've had a Windows Update the same day, however. Could you do a file system search inside `%LocalAppData%\Packages` for "MaxTo" or "DigitalCreations" or see if you come up with any hits? (This was the case with issue #415, where Windows created wrong "copies" of our DLLs, and it happened after a Windows update)

**Comment by gsrodrigostadler** [comment]
@vegardlarsen I found only the following file:
`AppData\Local\Packages\Microsoft.Windows.Search_cw5n1h2txyewy\LocalState\AppIconCache\100\com_squirrel_MaxTo_MaxTo_Core`
found no file for DigitalCreations.

**Comment by vegardlarsen** [comment]
I think I have a solution to this issue. It involves downloading and placing these two files inside the MaxTo install directory `%LocalAppData%\MaxTo\app-2.1.5`. [Files are here](https://sky.vegard.name/s/WmiaaHo7NosgndS).

These were files we don't ship with MaxTo, because they previously weren't necessary. I guess that changed at some point.

After placing those files, delete the folder in `%LocalAppData%\grabacr.net` (if it exists)? It should force the virtual desktop library to regenerate the files correctly. 

Then start MaxTo to see if it works and report back here.

**Comment by Eugene847** [comment]
Dears, 
for me it works now with the new dll files vegardlarsen provide. 
Thanks for the great support. 

best regards

**Comment by vegardlarsen** [comment]
@Eugene847 Great. Then this will be the official workaround for now, until we can get a new version out which fixes the problem. I've made quite a bit of progress replacing the library that causes this issue in the last few days; with the downside that we may not support older versions of Windows than Windows 10 version 1709. I am thinking that this change will be in 2.2 and not a patch release for 2.1; as it is a bigger and riskier change.

I may at some point ask for the people in this thread to try out a preview build of that release so we can be sure that it works.

**Comment by BenDavidson90** [comment]
Hi, same bug here.
MaxTo 2.1.5 installed today, latest Windows 10 update (20H2 19042.630). The workaround with the 2 DLLs works fine.

**Comment by metronom72** [comment]
the same issue.

**Comment by vegardlarsen** [comment]
@BenDavidson90 @metronom72 Did you try the workaround above?

**Comment by shie-erlich** [comment]
workaround works for me!

**Comment by vegardlarsen** [comment]
This has been fixed for version 2.2.0.
